### PR TITLE
[tests-only][full-ci] Added schema for key and value pair assertion

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
Json schema assertion for suite `apiShareSpaces`. for remaining `match key and value pairs` steps.

- [ ] Forward port to master branch

### Related issue:
https://github.com/owncloud/ocis/issues/3179

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/shareSpaces.feature [hunk 1]
```diff
@@ -19,16 +19,36 @@ Feature: Share spaces
 
   Scenario Outline: A Space Admin can share a space to another user
     When user "Alice" shares a space "share space" with settings:
-      | shareWith | Brian  |
+      | shareWith | Brian   |
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesShares/shareSpaces.feature:22]:
```suggestion
      | shareWith | Brian  |
```

**Review by saw-jan — COMMENTED** [review]
others LGTM

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiSpacesShares-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/20853/47/1) failed. Further test are cancelled...

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 2]
```diff
@@ -1103,6 +1103,7 @@ public function theJsonDataFromLastResponseShouldMatch(
 	/**
 	 * @Then /^for user "([^"]*)" the JSON response of space project should match$/
 	 * @Then /^for user "([^"]*)" the JSON response should contain space called "([^"]*)" and match$/
+     * @Then /^for user "([^"]*)" the JSON response should contain space called "([^"]*)" (?:owned by|granted to) "([^"]*)" and match$/
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/SpacesContext.php:1106]:
```suggestion
	 * @Then /^for user "([^"]*)" the JSON response should contain space called "([^"]*)" and match$/
         * @Then /^for user "([^"]*)" the JSON response should contain space called "([^"]*)" (?:owned by|granted to) "([^"]*)" and match$/
```

**Review by saw-jan — APPROVED** [review]

**Pull request merged**